update-github-hooks-ip.py

```diff
@@ -16,5 +16,5 @@
 if cnt:
     system("mv %s.tmp %s" % (ip_file, ip_file))
 else:
-    system("rm -f %s.tmp %s" % ip_file)
+    system("rm -f %s.tmp" % ip_file)
```

Comment:
for cms-bot scripts which do not need py2/py3 compatibility, please remove `from _py2with3compatibility ...` statements and directly import what is needed for python3